package.json

```diff
@@ -14,7 +14,7 @@
     "postbuild": "copyfiles -u 1 src/server/views/**/* build && copyfiles -u 4 src/server/services/QrCodeService/assets/**/* build/server/services/QrCodeService",
     "start": "node build/server/index.js",
     "client-dev": "webpack-dev-server --mode development --host 0.0.0.0 --devtool inline-source-map --hot",
-    "server-dev": "ts-node-dev --respawn --transpileOnly --inspect=0.0.0.0 -- src/server/index.ts",
+    "server-dev": "ts-node-dev --poll --respawn --transpileOnly --inspect=0.0.0.0 -- src/server/index.ts",
```

Comment:
what does this do

Reply:
This might be something I left behind while I was working on it. It helps with the problem of ts-node-dev not restarting in networked environments. More information here: https://github.com/remy/nodemon#application-isnt-restarting

Reply:
to add, without the --poll flag, my local development environment also struggles to hot-reload, which is why i thought we could also just merge this change into develop at the same time

Reply:
the legacy watch mode counterpart in ts-node-dev is --poll